lib/cli/test/fixtures/angular-cli-v7/.editorconfig

```diff
@@ -0,0 +1,13 @@
+# Editor configuration, see http://editorconfig.org
```

Comment:
Were there ever changes to a fixture app? I removed .editorconfig in the v6 fixture because I thought it would just apply rules you don't want to apply 

But they are re-written by the pre-commit anyway, I guess

Reply:
I can delete this... didn't even notice it =)

Reply:
They added this file in the generator of `ng new` - I don't know why.